ssr/src/page/wallet/txn.rs

```diff
@@ -76,56 +84,130 @@ impl TxnTag {
+                                },
+                                TxnInfoType::Received { from } => Some(view! {<div class="text-sm md:text-md text-white/50">{format!("From: {}", from)}</div>}),
+                                TxnInfoType::Sent { to } => Some(view! {<div class="text-sm md:text-md text-white/50">{format!("To: {}", to)}</div>}),
+                                TxnInfoType::Transfer { from, to } => Some(view! {
+                                    <div class="flex flex-col space-y-1">
+                                    <div class="text-sm md:text-md text-white/50">{format!("From: {}", from)}</div>
+                                    <div class="text-sm md:text-md text-white/50">{format!("To: {}", to)}</div>
+                                    </div>
+                                })
+                            }
+                        }
+                    }
                 </div>
             </div>
+            <div class="flex flex-col top-0 text-right">
             <span class=move || {
-                if direction.positive() {
-                    "text-green-600 justify-self-end"
-                } else {
-                    "text-red-600 justify-self-end"
+                match direction {
+                    TxnDirection::Added => "text-green-600 font-semibold",
+                    _ => "text-white font-semibold",
                 }
-            }>{bal_res}COYNs</span>
+            }>{format!("{} {}", bal_res, symbol)}</span>
+            <span class="text-sm md:text-md text-white/50">
+                {
+                    DateTime::from_timestamp(
+                        (info.timestamp / 1_000_000_000) as i64,  // seconds
+                        ((info.timestamp % 1_000_000_000) / 1_000) as u32 // microseconds
+                    ).map(|dt| {
+                        format!(
+                            "{} {}, {} {:02}:{:02} {}",
+                            match dt.date.month {
+                                1 => "January",
+                                2 => "February",
+                                3 => "March",
+                                4 => "April",
+                                5 => "May",
+                                6 => "June",
+                                7 => "July",
+                                8 => "August",
+                                9 => "September",
+                                10 => "October",
+                                11 => "November",
+                                12 => "December",
+                                _ => unimplemented!()
+                            },
+                            dt.date.day,
+                            dt.date.year,
+                            if dt.time.hour > 12 { dt.time.hour - 12 } else { dt.time.hour },
+                            dt.time.minute,
+                            if dt.time.hour >= 12 { "PM" } else { "AM" },
+                        )
+                    }).ok()
```

Comment:
i would move this logic to https://github.com/yral-dapp/hot-or-not-web-leptos-ssr/blob/main/ssr/src/utils/time.rs